### PR TITLE
add Neoprom peers.txt

### PR DESCRIPTION
**Pull request by neoprom**
## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: celestia/peers.txt [hunk 1]
```diff
@@ -1 +1,2 @@
+b1187a51400cef8b49a8482088eac57327989fe8@135.181.75.114:26656
```
Comment (evan-forbes) [inline celestia/peers.txt:1]: please fix conflict by submitting the peer info in a separate file in this directory named as `your-validator.txt`

**Review by evan-forbes — CHANGES_REQUESTED** [review]

**Comment by neoprom** [comment]
this ok or better do new PR?

File: celestia/peers.txt [hunk 2]
```diff
@@ -1 +0,0 @@
-d0c530290267e1c539f27061c9446d5042eaf1ca@161.97.180.20:26656
```
Comment (evan-forbes) [inline celestia/peers.txt:1]: wait is the peer ID already in the peers.txt?

Comment (evan-forbes) [reply celestia/peers.txt:1]: we want to avoid conflicts in this file, and can do that by not changing it relative to main

Comment (neoprom) [reply celestia/peers.txt:1]: Not our peer and dont know why it was in there, we submitted only one see https://github.com/celestiaorg/networks/pull/382/commits/3038515473763345c2b5b781f0a2bdbefd9d7484

Comment (neoprom) [reply celestia/peers.txt:1]: If not ok i can close this PR and make new

Comment (evan-forbes) [reply celestia/peers.txt:1]: the important thing is to not modify the peers.txt file relative to the master branch. If that occurs, there will likely be a conflict. one way to make sure there isn't a change is to copy the file from the master branch and paste it in this one. That or just revert any changes to that file.

**Comment by neoprom** [comment]
this ok now? I change back the peers.txt and add neoprom.txt

**Comment by evan-forbes** [comment]
we can actually see if this is okay or not on github. There should be not changes to the peers.txt file. I would look up a video on youtube to learn about git and diffs is this is not making sense
Screenshot from 2023-10-24 18-08-39
Screenshot from 2023-10-24 18-08-22

**Comment by neoprom** [comment]
Ok i see. Problem is I am making changes here directly on Github web and no revert possible here. Would it be easiest to close this PR and just make new with neoprom.txt and peer inside?

**Comment by neoprom** [comment]
I hope it good now

File: celestia/peers.txt [hunk 3]
```diff
@@ -1,22 +1,8 @@
...
-95c00bb9f24e907891e9a98c4ad506af093a7944@87.106.237.243:26656
-4ad28c8a0b04bac4aac441b105c815c449e2182f@65.108.61.189:26656
-c7dac72f630d2b3c3354ed664bcd6704a8f9e627@51.79.229.177:26656
-ce2873ef3893f0da50eb27892761e9c06e704944@159.203.143.94:26656
-1b92a439555a19ee6edea85b184abb6873757dae@51.89.173.96:26656
-ea9d858371ba86ae8f9c5b292c105329029c9efd@5.199.174.19:26656
-5bc86308e2b3730ce611abf622c8be752780f934@5.199.172.53:26656
-acca7837e4eb5f9dc7f5a94ed1d82edda6931ff8@rpc-celestia.pops.one:26656
-f5bce79feec9068154e7f50fe8db7a8191a58663@82.220.38.204:10056
-629c36a551ee1a3c8bb3ecee6e00a9977a60e53e@65.108.46.248:56656
```
Comment (adlerjohn) [inline celestia/peers.txt:14]: Don't modify this file

**Comment by YazzyYaz** [comment]
@neoprom  i suggest closing this PR and making a new one.

Just add one file called neoprom.txt with your peer ID. don't modify peets.txt.

**Pull request closed without merging**